test/gcs-test.js

```diff
@@ -0,0 +1,153 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const random = require('bcrypto/lib/random');
+const assert = require('bsert');
+const BasicFilter = require('../lib/golomb/basicFilter');
+const {U64} = require('n64');
+const Golomb = require('../lib/golomb/golomb');
+
+const key = random.randomBytes(16);
+
+const contents1 = [
+  Buffer.from('Alex', 'ascii'),
+  Buffer.from('Bob', 'ascii'),
+  Buffer.from('Charlie', 'ascii'),
+  Buffer.from('Dick', 'ascii'),
+  Buffer.from('Ed', 'ascii'),
+  Buffer.from('Frank', 'ascii'),
+  Buffer.from('George', 'ascii'),
+  Buffer.from('Harry', 'ascii'),
+  Buffer.from('Ilya', 'ascii'),
+  Buffer.from('John', 'ascii'),
+  Buffer.from('Kevin', 'ascii'),
+  Buffer.from('Larry', 'ascii'),
+  Buffer.from('Michael', 'ascii'),
+  Buffer.from('Nate', 'ascii'),
+  Buffer.from('Owen', 'ascii'),
+  Buffer.from('Paul', 'ascii'),
+  Buffer.from('Quentin', 'ascii')
+];
+
+const contents2 = [
+  Buffer.from('Alice', 'ascii'),
+  Buffer.from('Betty', 'ascii'),
+  Buffer.from('Charmaine', 'ascii'),
+  Buffer.from('Donna', 'ascii'),
+  Buffer.from('Edith', 'ascii'),
+  Buffer.from('Faina', 'ascii'),
+  Buffer.from('Georgia', 'ascii'),
+  Buffer.from('Hannah', 'ascii'),
+  Buffer.from('Ilsbeth', 'ascii'),
+  Buffer.from('Jennifer', 'ascii'),
+  Buffer.from('Kayla', 'ascii'),
+  Buffer.from('Lena', 'ascii'),
+  Buffer.from('Michelle', 'ascii'),
+  Buffer.from('Natalie', 'ascii'),
+  Buffer.from('Ophelia', 'ascii'),
+  Buffer.from('Peggy', 'ascii'),
+  Buffer.from('Queenie', 'ascii')
+];
+
+describe('GCS', function() {
+  let filter1 = null;
+  let filter2 = null;
+  let filter3 = null;
+  const basicFilter = new BasicFilter();
+  const testFilter = new Golomb(25, new U64(786515));
+  let testFilter1 = null;
+  let testFilter2 = null;
+  let testFilter3 = null;
+
+  it('should test GCS filter build', () => {
+    filter1 = basicFilter.fromItems(key, contents1);
+    assert(filter1);
+    testFilter1 = testFilter.fromItems(key, contents1);
+    assert(testFilter1);
+  });
```

Comment:
add at least one test that proves that the two filters are not the same (diff params = diff filter)

Reply:
`assert(!filter1.equals(testFilter1))`